pkg/client/connector/userd_trafficmgr/traffic_manager.go

```diff
@@ -172,7 +173,10 @@ func (tm *trafficManager) Run(c context.Context) error {
 		InstallId: tm.installID,
 		Product:   "telepresence",
 		Version:   client.Version(),
-		ApiKey:    func() string { tok, _ := tm.callbacks.GetCloudAPIKey(c, "manager", false); return tok }(),
+		ApiKey: func() string {
+			tok, _ := tm.callbacks.GetCloudAPIKey(c, a8rcloud.KeyDescTrafficManager, false)
```

Comment:
Not that it changes in this PR, but ignoring the error from `GetCloudAPIKey()` seems wrong.

Reply:
Including an apikey with this request is optional; we might not even be logged in.